src/Model.js

```diff
@@ -300,8 +299,12 @@ class Model {
     }
 
     const response = this.store.fetch(url, { method, body })
+    const result = this.store.updateRecords(response, this)
+    Object.defineProperties(result, {
+      isInFlight: { value: this.isInFlight }
+    })
```

Comment:
do we need this?

Reply:
I thought that this was the one thing we wanted to keep? Can you elaborate on why you think we might not need it?

Reply:
I don't think it's actually setting anything.